encoder-brave/README.md

```diff
@@ -0,0 +1,8 @@
+# encoder-otel-brave
```

Comment:
```suggestion
# encoder-brave
```

Reply:
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/9a895b742293dd28a42eb9a162272788c452998e